media/css/mozorg/contribute/friends.less

```diff
@@ -28,188 +47,372 @@
         }
 
         h1 {
-            .font-size(4rem);
+            .font-size(64px);
+            margin-bottom: @baseLine * 2;
             text-shadow: 1px 1px 2px #333;
 
             @media only screen and (max-width: 790px) {
-                .font-size(3rem);
+                .font-size(48px);
+            }
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                .font-size(40px)
             }
         }
         p {
-            margin: 0 auto;
-            max-width: 610px;
             .font-size(1.4rem);
+            margin: @baseLine auto 0;
+            max-width: 700px;
             text-shadow: 1px 1px 2px #333;
 
             @media only screen and (max-width: 790px) {
+                .font-size(18px);
                 max-width: 500px;
-                .font-size(1.2rem);
             }
-        }
-    }
-    .ff-join-now {
-        display: block;
-        background-color: #f26c24;
-        color: #fff;
-        margin: 70px auto 0 auto;
-        padding: 10px 30px;
-        border: 0;
-        border-radius: 6px;
-        width: 200px;
-        .font-size(1.2rem);
 
-        @media only screen and (max-width: @breakMobileLandscape) {
-            margin-top: 35px;
+            @media only screen and (max-width: @breakMobileLandscape) {
+                .font-size(@largeFontSize);
+            }
         }
 
-        &:hover {
-            text-decoration: none;
+        .ff-join-now {
+            margin: 70px auto 0 auto;
+            transition: opacity 0.3s ease-in 0.1s;
+
+            &.invisible {
+                cursor: default;
+                opacity: 0;
+            }
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                margin-top: 35px;
+            }
         }
     }
+
     main {
         background-color: #06a6e0;
     }
+
     .ff-body-content {
         color: #fff;
         margin: 0 auto;
-        padding: 75px 180px;
+        max-width: 1000px;
+        padding: 75px 0;
         text-align: center;
-        max-width: 1400px;
-
-        @media only screen and (max-width: 1100px) {
-            padding: 75px 100px;
-        }
-
-        @media only screen and (max-width: 790px) {
-            padding: 75px 20px;
-        }
 
         h2 {
             margin-bottom: 50px;
+            text-shadow: none;
 
             @media only screen and (max-width: @breakMobileLandscape) {
                 margin-bottom: 40px;
             }
         }
-        .how {
-            display: inline-block;
-            padding-left: 5%;
-            width: 27%;
-            vertical-align: top;
+
+        h3 {
+            .font-size(1.2rem);
+            text-shadow: none;
+            text-transform: uppercase;
+
+            @media only screen and (max-width: 790px) {
+                .font-size(1rem);
+                line-height: 1.5;
+            }
+        }
+
+        .how-wrapper {
+            .flexbox();
```

Comment:
This actually doesn't look too bad in IE. On the versions that actually load the CSS (8 and up), only 10 looks a little bad. 8 and 9 just stack, and 11 renders correctly. 10 renders the items side by side, just all left justified.

I'm okay with the above, but I'm pretty much always okay with older browsers not being perfect. :grin: